backends/advanced/src/advanced_omi_backend/routers/modules/system_routes.py

```
@@ -128,3 +128,41 @@ async def get_enrolled_speakers(current_user: User = Depends(current_active_user
 async def get_speaker_service_status(current_user: User = Depends(current_superuser)):
     """Check speaker recognition service health status. Admin only."""
     return await system_controller.get_speaker_service_status()
+
+
+# Memory Configuration Management Endpoints
+
+@router.get("/admin/memory/config/raw")
+async def get_memory_config_raw(current_user: User = Depends(current_superuser)):
```

Ruff (0.12.2)

Unused function argument: `current_user`

(ARG001)